pkg/collector/dist/datadog.yaml

```diff
@@ -136,6 +136,22 @@ metadata_collectors:
 
 # Autodiscovery
 #
+# Change the root directory to look at to get cgroup statistics. Useful when running inside a
+# container with host directories mounted on a different folder.
+# Default if environment variable "DOCKER_DD_AGENT" is set to "yes"
+# "/host/sys/fs/cgroup" and "/sys/fs/cgroup" if not.
+#
+# Example for the docker-dd-agent container:
```

Comment:
can we just remove this line? The exemple above is enough, and it makes it look like the following line is just an example when it's actually the line to uncomment and modify.